README.md

```diff
@@ -102,19 +105,32 @@ there, please contact support for Windows driver information.
   - Download the ZIP file, extract, and run `SetupSerialIO.exe`
 - [Intel Thunderbolt Driver](./drivers/tbt_dch_sw_rev77.zip)
   - Download the ZIP file, extract, and run `TbtWosRelease_77/Thunderbolt(TM) Software Installer.exe`
-- [Intel WiFi Driver](./drivers/wifi-phwfw01690_21.110.1.1_g.zip)
+- [Intel WiFi Driver](./drivers/wifi-PHWFW06454_22.150.0.3_G.zip)
   - Download the ZIP file, extract, and run `UWD/Win64/Installer/WirelessSetup.exe`
 - [Realtek Card Reader](./drivers/RtsXStor_10.0.370.188.zip)
   - Download the ZIP file, extract, and run `RtsXStor_10.0.370.188/setup.exe`
 - Other drivers are installed automatically through Windows Update
 
+### Galago Pro (galp6)
+
+- [Intel Chipset Driver](./drivers/Chipset-10.1.19159.8331-Public-MUP.zip)
+  - Download the ZIP file, extract, and run `SetupChipset.exe`
+- [Intel HID Event Filter Driver](./drivers/hideventfilterdriver-2.2.1.386_rs5_19h1_20h1_21h2_certified.zip)
+  - Download the ZIP file, extract, and run `Installer/Setup.exe`
+- [Intel SerialIO Driver](./drivers/serialio_30.100.2031.2_pv_adl-lp_22h2.zip)
```

Comment:
Like Darp8, the SerialIO driver reports that the platform isn't supported, and the trackpad does not work as far as I can tell without it.